types/auth0/index.d.ts

```diff
@@ -379,6 +382,27 @@ export type ClientGrant = Partial<CreateClientGrant> & {
   id?: string;
 };
 
+export interface GetClientGrantsOptions {
+per_page?: number;
+  page?: number;
+  /**
+   * The audience.
+   */
+  audience?: string;
+  /**
+   * The id of the client (application).
+   */
+  client_id?: string;
+}
+
+export interface GetClientGrantsOptionsPaged extends GetClientGrantsOptions {
+  include_totals?: boolean;
```

Comment:
```suggestion
  /**
   * true if a query summary must be included in the result, false otherwise
   * @default false
   */
  include_totals?: boolean;
```
https://github.com/auth0/node-auth0/blob/7bc4a43cfcdbd32828a6a22b2e299770c55345d5/src/management/index.js#L714